bigtable/integration_test.go

```diff
@@ -2142,6 +2144,63 @@ func TestIntegration_AdminBackup(t *testing.T) {
 	}
 }
 
+// Blackhole directpath address to test fallback.
+func TestIntegration_DirectPathFallback(t *testing.T) {
+	ctx := context.Background()
+	testEnv, _, _, table, _, cleanup, err := setupIntegration(ctx, t)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer cleanup()
+
+	if err := populatePresidentsGraph(table); err != nil {
+		t.Fatal(err)
+	}
+
+	// Precondition: wait for DirectPath to connect.
+	countEnough := exerciseDirectPath(ctx, testEnv, table /*blackholeDP = */, false)
```

Comment:
Go style does not have inline comments like this -- it's usually a sign the function being called should be split into more than one function. Perhaps one for the `true` case and one for the `false` case. That would also hopefully make it easier to understand & debug.

Reply:
Sorry I do not understand this comment. I found this test https://github.com/googleapis/google-cloud-go/blob/master/bigtable/integration_test.go#L2013 also have similar inline comments. So could you give me an example about how should I write comments? Thanks for help!